include/revng/Model/CommonTypeMethods.h

```diff
@@ -178,12 +178,21 @@ class CommonTypeMethods {
   /// This helper returns true if and only if this type is a primitive or
   /// a pointer.
   ///
+  /// \note This helpers considers enums to be primitives.
+  ///
   /// \note it asserts if called on something that's not an object.
   ///
   /// \note similarly to other `isX` methods, this also skips typedefs.
   bool isScalar() const;
 
-  // TODO: we want more of these, like `isAggregate`
+  /// This helper returns true if and only if this type is is not scalar.
+  ///
+  /// \note it asserts if called on something that's not an object.
+  ///
+  /// \note similarly to other `isX` methods, this also skips typedefs.
+  bool isAggregate() const { return !isScalar(); }
+
+  // TODO: we want more of these.
```

Comment:
At this point this comment is a bit vague, what else? I'd drop.

Reply:
Done.